controllers/utils.go

```diff
@@ -51,41 +53,46 @@ func GeneratePassword(length int) []byte {
 	return buf
 }
 
-func GetBuild(ctx context.Context, req ctrl.Request, workflowID string, r *KogitoServerlessWorkflowReconciler) (apiv08.KogitoServerlessBuild, error) {
+func GetBuild(ctx context.Context, req ctrl.Request, workflowID string, r *KogitoServerlessWorkflowReconciler, log logr.Logger) (apiv08.KogitoServerlessBuild, error) {
 	buildInstance := &apiv08.KogitoServerlessBuild{}
 	buildInstance.Spec.SwfName = workflowID
 	error := r.Client.Get(ctx, types.NamespacedName{Namespace: req.NamespacedName.Name, Name: workflowID}, buildInstance)
 	return *buildInstance, error
 }
 
-func CreateBuild(ctx context.Context, workflowID string, workflowSwf []byte, r *KogitoServerlessWorkflowReconciler) (apiv08.KogitoServerlessBuild, error) {
+func CreateBuild(ctx context.Context, workflowID string, workflowSwf []byte, r *KogitoServerlessWorkflowReconciler, log logr.Logger) (apiv08.KogitoServerlessBuild, error) {
```

Comment:
Why are you adding the logger here if it's not being used? I liked the Camel-K approach we replicated in the builder, where we implemented `InjectLogger` and delegated it to the caller to inject it. This way, we avoid passing too many arguments. What about considering this approach here?